### DISCUSSION TITLE
Colored Books

### DISCUSSION THREAD

**Discussion by mibby** [PurpurMC/Purpur#1084] [__Completed__] [answered]
Similar to the colored anvil feature when renaming items in an anvil, would it be possible to add an option to allow translating formatting codes (legacy & hex) in books to colors?

i.e. https://www.spigotmc.org/resources/colored-books.78355/ but integrated instead of plugin dependent

**Comment by RhythmicSys** [comment, marked as answer]
This is already an option, usable through the [`purpur.book.color.sign`](https://purpurmc.org/docs/Permissions/#purpurbookcolorsign) permission

**Comment by mibby** [comment]
Ah, I didn't know this exists already! I was looking at the config looking for an option. 

Shame you can't preview HEX but does seem to work great. Thanks for the info!

**Comment by encode42** [comment]
There's actually a mod that does just that! (I believe)

https://modrinth.com/mod/stendhal
